src/js/components/DataFilter/DataFilter.js

```diff
@@ -227,6 +227,8 @@ export const DataFilter = ({
                 : undefined
             }
             onClose={() => setSearchText('')}
+            labelKey="label"
+            valueKey={{ key: 'value', reduce: true }}
```

Comment:
Should we only pass these if options is an array of objects?

Reply:
Good question -- I had thought about the same, but tested it out with just an array of strings and didn't see any issues with functionality. This is because the logic inside SelectMultiple will check if the option is an object or not.

Let me know if you're comfortable with leaving as is. Otherwise, I could add a check to see if the first option is an object and only pass the props through in that case.

Reply:
Sounds good, just wanted to verify that the code in SelectMultiple is checking that options is an object before trying to use labelKey or valueKey. I think it is fine to leave as is